services/discordService.js

```diff
@@ -7,22 +7,17 @@ const getDiscordMembers = async () => {
     algorithm: "RS256",
     expiresIn: config.get("rdsServerlessBot.ttl"),
```

Comment:
- NIT PICK: Moving the places we do `config.get(<config>)` into its own variable would make this code look much cleaner. 
    - This is optional not a P0.

Reply:
Will cover this in the coming PR as that service was written earlier, to modify it I would raise a new PR. (Focusing on the delta minimization of the current PR).